lib/config/index.js

```diff
@@ -88,6 +88,11 @@ config.isStandardHTTPPort = (function isStandardHTTPPort () {
   return !config.useSSL && config.port === 80
 })()
 
+// Use HTTPS protocol if the internal TLS server is enabled
+if (config.useSSL === true) {
+  config.protocolUseSSL = true
```

Comment:
As I wrote in the matrix chat, do we want to raise a warning if `protocolUseSSL` is explicitly set to false?

Would be something like this:

```suggestion
  if (config.protocolUseSSL === false) {
    logger.warn('You configured protocolUseSSL to false, but this will be overriden as you set useSSL to true.')
  }
  config.protocolUseSSL = true
```

Reply:
I added a warning